### PR TITLE
Updating dataset for 4.16.24 version

### PR DESCRIPTION
**Pull request by SamStruthers**
Work:
Initially this data was housed in two separate datasets (CPF, PWQN), as these projects become more and more similar it is more inconvenient and not really necessary to differentiate them. As such, I am merging them into a single repo which will be used to update the zenodo pub. This repo has ~ 0 analysis, it is only created to be a package to send to zenodo for publishing to external collaborators or funders. 

Review requested:
- imagine you are a totally new person to the project (or some rando who loves WQ data)
- Download the whole repo, read the READMEs and play with the cleaned dataset
- Is there anything that doesn't make sense or any clarification that should be added to README's?

### CODE REVIEW

File: scripts/01_chem_prep.qmd
Comment (juandlt-csu) [inline scripts/01_chem_prep.qmd:1]: question: are users expected to interact with this script, or for that matter any of the other scripts?

File: scripts/demo.R
Comment (juandlt-csu) [inline scripts/demo.R:1]: question: would this pull data that is different from the data in this repository?

File: data/raw/ROSS_20240423.xlsx
Comment (juandlt-csu) [inline data/raw/ROSS_20240423.xlsx:1]: question: What is the reason for having .xlsx and .csv files in the repo?

**Review by juandlt-csu — APPROVED** [review]
The readme is clear about where to find the data in the repository and what the different files mean. I was able to load in data and play with it.

I would recommend making a "Code Organization" section and put the script info there instead of under "Data Organization". I'm not entirely sure what is needed to hit the project goals, but this seems alright to me.

File: scripts/01_chem_prep.qmd
Comment (SamStruthers) [reply scripts/01_chem_prep.qmd:1]: Added a note in the code section to address this

**Pull request merged**